azure-proto-compute/Extensions/AzureSubscriptionExtension.cs

```diff
@@ -45,14 +45,14 @@ public static AsyncPageable<VirtualMachine> ListVirtualMachinesAsync(this Subscr
 
         public static Pageable<ArmResource> ListVirtualMachinesByName(this SubscriptionOperations subscription, ArmSubstringFilter filter, int? top = null, CancellationToken cancellationToken = default)
         {
-            ArmFilterCollection filters = new ArmFilterCollection(VirtualMachineData.ResourceType);
+            ArmFilterCollection filters = new ArmFilterCollection(VirtualMachineOperations.ResourceType);
```

Comment:
What do you think of the idea of placing ResourceType on VirtualMachine?  Seems more natural.

Reply:
I have that as a backlog item to consider